docs/technical-overview/feature-flags.md

```diff
@@ -79,6 +79,20 @@ To enable such a feature:
 
 ![A screenshot of the Admin Feature Flag Panel](/img/docs/admin_feature_flags.png)
 
+Note that you made need to add a feature before enabling it.
+
+## Adding a feature flag
+
+To add a feature flag, you must be on the `/admin/feature_flags` page and click "Add Feature".
+
+![A screenshot of the "Add Feature" button](https://user-images.githubusercontent.com/16007075/184420699-4917228c-fad8-49ea-b68b-6b20db972e43.png)
+
+Next, input the name of the feature and follow the instructions on screen, then click "Add Feature".
+
+![Screen Shot 2022-08-12 at 2 25 26 PM](https://user-images.githubusercontent.com/16007075/184421097-54d65918-cff1-4704-97ad-7ed408d988f1.png)
+
+Navigate back to `/admin/feature_flags` to enable your feature.
```

Comment:
Configure your feature flag (if required) before clicking on "Fully enable".

![A screenshot of the configuration screen for adding a feature flag](https://user-images.githubusercontent.com/2786819/184673662-d94fcbd7-26b1-472d-8ad9-e15495bfd32d.png)

When you navigate back to `/admin/feature_flags`, you should see a green dot next to your newly added feature flag.